sys/shell/commands/sc_gnrc_ipv6_nib.c

```diff
@@ -57,16 +57,16 @@ static void _usage_nib_neigh(char **argv)
     printf("usage: %s %s [show|add|del|help]\n", argv[0], argv[1]);
     printf("       %s %s add <iface> <ipv6 addr> [<l2 addr>]\n", argv[0], argv[1]);
     printf("       %s %s del <ipv6 addr>\n", argv[0], argv[1]);
-    printf("       %s %s show <ipv6 addr>\n", argv[0], argv[1]);
+    printf("       %s %s show [ipv6 addr]\n", argv[0], argv[1]);
```

Comment:
This is supposed to be `[iface]`. Would you be so kind and change this as well?

Reply:
Ack

Reply:
Done